charts/alfresco-search-service/values.yaml

```diff
@@ -64,12 +64,16 @@ ingress:
 repository:
   # -- Alfresco repository URL
   url: http://alfresco-search-service/solr
+  # -- repository seucurity level to use when tracking the repo ('none' or 'secret')
+  securecomms: secret
```

Comment:
I wonder if we really want to mimic the app behavior here or at least provide a more meaningful name for this key, or we could also provide a boolean instead which internally map to `none` or `secret`?

Reply:
We can rule out the boolean as there is actually 3 possible values (`none`, `secret` & `https`).
As for changing the name I'm not sure that's a good idea as the meaning differs based on the search engine. `https` for solr means mTLS while for elasticsearch it can be simply encrypted http. Sticking to what people already know might be less confusing for them.